### PR TITLE
TypeFormPopup component

### PR DESCRIPTION
**Pull request by blackinno**
Closed. #83

### CODE REVIEW

**Review by EmmanuelDrouin — COMMENTED** [review]
@blackinno looks good to me, just one thing, can you add the `data-submit-close-delay="0"` ? the popup should close directly after the completion of the form.

**Comment by EmmanuelDrouin** [comment]
Looks good to me

File: packages/TypeFormPopup/TypeFormPopup.vue [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<template>
+  <a @click="onClickedPopup">
```
Comment (antho1404) [inline packages/TypeFormPopup/TypeFormPopup.vue]: If we put an `a` tag in the slot then we will have a link in a link which I'm not sure is something html valid.

Maybe we can just have `<slot @click="onClickedPopup" />`

Comment (blackinno) [reply packages/TypeFormPopup/TypeFormPopup.vue]: I have changed from `a` tag to `div` tag  because we can't use `slot` directly in `template`

**Comment by EmmanuelDrouin** [comment]
@blackinno we need to import the "Theme" in the component like all other components, otherwise looks good to me

**Comment by blackinno** [comment]
@EmmanuelDrouin, Done, I add a `theme dependency` and import `theme variables`.

**Comment by antho1404** [comment]
@EmmanuelDrouin why do you want to import the theme? We don't use it so no need to import it.
@blackinno there is some useless css.
Why do we have a click on a div? Is it really mandatory? can't we use the slot directly and if not why?

**Comment by blackinno** [comment]
@antho1404, It's mandatory. We can't use `slot` as a component root element because it may contain multiple nodes also `slot` doesn't work on an event like `@click` or other events.
Do you have any suggestions element to use?

**Comment by EmmanuelDrouin** [comment]
Why do you guys remove the a:hover in the style? now we have the same issue as before, on hover on the button or the link the cursor don't change to a pointer.

**Comment by EmmanuelDrouin** [comment]
@antho1404 Wanted to import the Theme because before, the default text example on the storie didn't have the right font and size.

**Comment by antho1404** [comment]
the theme on the example in the storybook is the theme for the storybook only, maybe the storybook theme should be better but the component should not include the theme because you want it in the storybook.

The css was removed because this is not the responsibility of the components again but the storybook,  at least to put it on the `a` (the component doesn't have an `a` tag).

File: packages/TypeFormPopup/TypeFormPopup.vue [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+<script>
+export default {
+  name: "TypeFormPopup",
+  props: {
+    id: { type: String, required: true },
+    url: { type: String, default: "https://mesgfoundation.typeform.com/to" }
+  },
+  data() {
+    return {
+      openPopup: undefined
```
Comment (antho1404) [inline packages/TypeFormPopup/TypeFormPopup.vue]:
```suggestion
      openPopup: null
```

**Review by antho1404 — APPROVED** [review]

**Pull request merged**